src/main/java/org/kiwiproject/dropwizard/util/metrics/ServerLoadFetcher.java

```diff
@@ -54,9 +52,8 @@ public ServerLoadFetcher() {
     public Optional<String> get() {
         var process = processes.launch("uptime");
 
-        try (var reader = new BufferedReader(new InputStreamReader(process.getInputStream(), UTF_8))) {
-            var output = reader.readLine();
-
+        try {
+            var output = KiwiIO.readInputStreamOf(process);
```

Comment:
Does the test take into account that there is a trailing newline?

Reply:
This seems to work.  I tested it with a newline